tools/Dockerfile

```diff
@@ -3,21 +3,19 @@ MAINTAINER "Stef Walter" <stefw@redhat.com>
 
 RUN yum -y update && yum clean all
 
-# A repo where we can find recent Cockpit builds for F21
+# A repo where we can find recent Cockpit builds for Fedora
 ADD cockpit-preview.repo /etc/yum.repos.d/
 
 # If there are rpm files in the current directory we'll install those,
 # otherwise use cockpit-preview repo. The Dockerfile is a hack around
 # Dockerfile lack of support for branches
-ADD *.rpm Dockerfile /tmp/
+ADD cockpit-ws-*.rpm Dockerfile /tmp/
 
 # Again see above ... we do our branching in shell script
-RUN cd /tmp && ( ls *.rpm > /dev/null 2> /dev/null && yum -y install cockpit*.rpm || yum -y --enablerepo=cockpit-preview install cockpit ) && yum clean all && rm -f /tmp/cockpit*.rpm
+RUN cd /tmp && ( ls *.rpm > /dev/null 2> /dev/null && yum -y install *.rpm || yum -y --enablerepo=cockpit-preview install cockpit-ws ) && yum clean all && rm -f /tmp/*.rpm
```

Comment:
we should split this into multiple lines
https://docs.docker.com/articles/dockerfile_best-practices/